packages/odyssey-storybook/.storybook/components/MuiThemeDecorator.tsx

```diff
@@ -16,20 +15,22 @@ const styles = {
 
 const odysseyTheme = createOdysseyMuiTheme({ odysseyTokens });
 
-export const MuiThemeDecorator: Decorator = (Story) => (
-  <OdysseyThemeProvider>
-    <OdysseyTranslationProvider>
-      {/* @ts-expect-error type mismatch on "typography" */}
-      <StorybookThemeProvider theme={odysseyTheme}>
-        <Fragment>
+export const MuiThemeDecorator: Decorator = (Story, context) => {
+  console.log({ context });
+
+  return (
+    <OdysseyThemeProvider>
+      <OdysseyTranslationProvider languageCode={context.globals.locale}>
+        {/* @ts-expect-error type mismatch on "typography" */}
+        <StorybookThemeProvider theme={odysseyTheme}>
           <CssBaseline />
           <div style={styles}>
             <ScopedCssBaseline>
               <Story />
             </ScopedCssBaseline>
           </div>
-        </Fragment>
-      </StorybookThemeProvider>
-    </OdysseyTranslationProvider>
-  </OdysseyThemeProvider>
-);
+        </StorybookThemeProvider>
+      </OdysseyTranslationProvider>
+    </OdysseyThemeProvider>
+  );
+};
```

Comment:
Now that we've inlined the context, we can actually make this an implicit return function again.

We don't need that `console.log` either ;P.

Reply:
Ahhh, whoops. Will remove. And can implicitly return as well. Good catch